indico/modules/groups/controllers.py

```diff
@@ -140,3 +141,10 @@ class RHGroupDeleteMember(RHLocalGroupBase):
     def _process(self):
         self.group.members.discard(User.get(request.view_args['user_id']))
         return jsonify(success=True)
+
+
+class RHGroupSearch(RHProtected):
+    def _process(self):
```

Comment:
Maybe use `webargs` here to do some basic sanitization?

Reply:
oh, I forgot about it..